grammars/clojure.cson

```diff
@@ -139,9 +139,11 @@
     'beginCaptures':
       '1':
         'name': 'punctuation.section.map.begin.clojure'
-    'end': '(\\})'
+    'end': '(\\}(?=[\\}\\]\\)\\s]*(?:;|\\n)))|(\\})'
     'endCaptures':
       '1':
+        'name': 'punctuation.section.map.end.clojure.paren-trail'
```

Comment:
Can we do something like `punctuation.section.map.end.trailing.clojure` instead?  The language should always be the most specific scope.